cfme/tests/networks/test_tag_tagvis.py

```diff
@@ -59,6 +59,8 @@ def child_visibility(appliance, network_provider, relationship, view):
 
 @pytest.mark.parametrize("relationship,view", network_test_items,
                          ids=[rel[0] for rel in network_test_items])
+@pytest.mark.uncollectif(
+    lambda relationship, appliance: "Load Balancers" in relationship and appliance.version > "5.11")
```

Comment:
As we confused like bug or feature; @lina-nikiforova I think better to add some uncollect `reason` like
```
@pytest.mark.uncollectif(
    lambda relationship, appliance: "Load Balancers" in relationship and appliance.version > "5.11",
    reason="<some reason like load balancers  not supported from 511>"
)
```

Reply:
*Required* - thanks!

Reply:
@digitronik absolutely!